2025-09-23 16:24:54 - root - INFO - [logging_config.py:51] - Logging configurado. Arquivo: C:\Users\Pedro Diniz\Documents\CadVision\CadVision_app\backend\logs\cadvision_20250923_162450.log
2025-09-23 16:24:54 - app - INFO - [logging_config.py:68] - EVENT: startup - DATA: {'timestamp': '2025-09-23T16:24:54.569577', 'service': 'app', 'event': 'startup', 'data': {'version': '1.1.0'}}
2025-09-23 16:24:54 - app.database - INFO - [database.py:175] - Banco de dados inicializado com sucesso (com novo schema).
2025-09-23 16:24:54 - app - INFO - [logging_config.py:68] - EVENT: database_initialized - DATA: {'timestamp': '2025-09-23T16:24:54.573593', 'service': 'app', 'event': 'database_initialized', 'data': {}}
2025-09-23 16:25:03 - app - INFO - [logging_config.py:68] - EVENT: shutdown - DATA: {'timestamp': '2025-09-23T16:25:03.310495', 'service': 'app', 'event': 'shutdown', 'data': {}}
2025-09-23 16:25:03 - main - INFO - [main.py:48] - Encerrando aplicação CadVision API.
